inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/TubeSink.java

```diff
@@ -285,7 +285,7 @@ public Status process() throws EventDeliveryException {
                     cachedMsgCnt.incrementAndGet();
                     DataProxyMetricItem metricItem = this.metricItemSet.findMetricItem(dimensions);
                     metricItem.readSuccessCount.incrementAndGet();
-                    metricItem.readFailSize.addAndGet(event.getBody().length);
+                    metricItem.readSuccessSize.addAndGet(event.getBody().length);
                 } else {
                     tx.rollback();
                     //logger.info("[{}] Channel --> Queue(has no enough space,current code point) "
```

Comment:
Could you please remove those unused log codes at the same time?

Reply:
I see a lot of such useless comments and codes. It is better to submit a separate issue to solve it.
@healchow